srv/notifyToConsole.js

```diff
@@ -1,34 +1,29 @@
 const NotificationService = require('./service');
 const cds = require("@sap/cds");
 const LOG = cds.log('notifications');
-const { buildNotification, doesKeyExist } = require("./../lib/utils");
 
 module.exports = class NotifyToConsole extends NotificationService {
   async init() {
-    // call NotificationService's init
-    await super.init();
-  }
-
-  notify() {
+    await super.init()
+    this.on("notification", req => {
+      const notification = req.data; if (!notification) return
+      LOG.info(notification)
```

Comment:
in audit-logging, we really use `console`. should we use cds.log there instead?

Reply:
No. `console` is actually better in this case → will change it here accordingly